### ISSUE TITLE
Hyperspy Logo SVG

### ISSUE THREAD

**Issue by CSSFrancis** [hyperspy/hyperspy#3292]
I was wondering if we could clean up the hyperspy logo a bit? It would be nice to have a `.svg` version of the logo that was a little bit higher of a resolution.

I played around with it a little bit:

 
![hyperspy](https://github.com/hyperspy/hyperspy/assets/41125831/7c79f411-1685-41ee-93b9-d6af574bdbc4)

**Comment by jlaehne** [comment]
Great idea, I was already annoyed at the mediocre quality when putting it in presentations. The good thing with a true svg is that we can have it at any resolution. The lower left corner is odd though. In fact, I would propose to keep the perspective and colors though for the hypercube and simply retrace the current logo as svg.

Concerning the version with text, we can indeed find a better font. Either the one you use on the mockup for the new website (looks like the numpy one), or the one from RosettaSciIO.

**Comment by CSSFrancis** [comment]
> Great idea, I was already annoyed at the mediocre quality when putting it in presentations. The good thing with a true svg is that we can have it at any resolution. The lower left corner is odd though. In fact, I would propose to keep the perspective and colors though for the hypercube and simply retrace the current logo as svg.

Yea I noticed that after I posted it. I'll mock up a couple of things. Retracing the original as well 
> 
> 
> Concerning the version with text, we can indeed find a better font. Either the one you use on the mockup for the new website (looks like the numpy one), or the one from RosettaSciIO.

That would be a good idea. I've been using noto (I think that is the rosettasciio font) and it looks pretty good.

**Comment by CSSFrancis** [comment]
I've done a couple of more mockups (using lato as the font)

I tried to trace the old logo and then Inkscape crashed. There are some lines that are very close to each other in the old perspective which are a little better in the new perspective.  If people really want me to I can copy the old one. I also cleaned up the colors a little bit as there are lots of colors in the older version and I think it makes more sense to have 4 main colors in the design that we can use as a standard. 

There might be some weird opacity things going on as well, I think we should decide if we want to make the colors on the logo a little more opaque.  It helps to make things a little less busy which I like.

What I Used (pulled from the old design):
-----------------------------------------
Green: 119347bf
Yellow: adc221df
Red: be4c4dbf
Purple:826e8fdf

Take my colors with a grain of salt, however, as I am color blind :) 

Light Design:
--------------
![hyperspy](https://github.com/hyperspy/hyperspy/assets/41125831/4b6f28ce-4aa8-4bc5-889a-3b0b0868c2d9)

Dark Design:
-------------
![hyperspy_white](https://github.com/hyperspy/hyperspy/assets/41125831/0aa9077e-6f4a-42a3-a688-c628f5083de7)

Light Banner:
--------------
![hyperspy_light_banner](https://github.com/hyperspy/hyperspy/assets/41125831/0f98d37a-b7c9-4dd1-bb0d-ee61c53ae903)

Dark Banner:
-------------
![hyperspy_white_banner](https://github.com/hyperspy/hyperspy/assets/41125831/66ef6b14-836a-484e-87c5-ee45bed0ae72)

**Comment by jlaehne** [comment]
Looks good. Indeed, I like the font in this context. In the banner, the hypercube could be offset a little more. Though I also kinda liked the 'superscript' hypercube in some of your earlier mockups. In the dark version, the background lines could be a bit lighter. Concerning the perspective and colors, I think we should get a broader discussion, in particular including @francisco-dlp and @ericpre

**Comment by CSSFrancis** [comment]
@jlaehne yes I think a bigger discussion about maybe updating the logo is a good idea.

I'll try to make a copy of the original design again just so we can compare.
